app/CurationExporter.php

```diff
@@ -50,39 +97,77 @@ protected function buildQuery($params)
+                'curations.created_at as Created',
+                'curations.gdm_uuid as GCI UUID',
+            ]
+        );
+
+        if (config('app.debug') && !app()->environment('testing')) {
+            array_unshift($columns, 'curations.id as ID');
+        }
+
+        return $columns;
+    }
+
+    private function buildStatusSubQuery()
+    {
+        $statusColumns = ['curation_id', 'name',];
+        $aggregateColumns = ['curation_id'];
+        $this->curationStatuses->each(function ($status) use (&$statusColumns, &$aggregateColumns, &$statusDateColumns) {
+            $statusColumns[] = DB::raw("CASE WHEN curation_status_id = {$status->id} THEN DATE(status_date) ELSE NULL END as `{$status->name}`");
+            $aggregateColumns[] = DB::raw("MAX(`{$status->name}`) as `{$status->name} Date`");
+        });
+
+        $statusDateQuery = DB::table('curation_curation_status')
+            ->select($statusColumns)
+            ->join('curation_statuses', 'curation_curation_status.curation_status_id', '=', 'curation_statuses.id')
+            ->whereIn(
+                DB::Raw('(curation_id, status_date, curation_status_id)'),
+                function ($query) {
+                    $query->select('curation_id', DB::raw('max(status_date) as status_date'), 'curation_status_id')
+                        ->from('curation_curation_status')
+                        ->groupBy('curation_id', 'curation_status_id');
+                }
+            );
+
+        return DB::query()
+            ->from($statusDateQuery, 'status_date_query')
+            ->select($aggregateColumns)
+            ->groupBy('curation_id');
+    }
+
     /**
      * @SuppressWarnings(PHPMD.ElseExpression) // used as callback in this class
      */
     public function getCsv($params = [], $csvPath = null)
     {
         $query = $this->buildQuery($params);
-
+       
         $path = $csvPath ?? $this->buildFileName($params);
 
+        $header = array_keys((array)$query->first());
         $fh = fopen($path, 'w');
-        fputcsv($fh, $this->buildHeader());
-        $query->lazy()->each(function ($curation) use ($fh) {
-            fputcsv($fh, $this->buildLine($curation));
+        fputcsv($fh, $header);
+        $query->lazy()->each(function ($row) use ($fh) {
+            fputcsv($fh, (array)$row);
         });
         fclose($fh);
 
         return $path;
     }
 
-    /**
-     * Filters $curation's statuses entries to the latest (by status_date) for each status
-     * b/c we only want the latest instance of a status for the curtion export.
-     */
-    private function getLatestStatusDates($curation)
-    {
-        return $curation->statuses
-        ->groupBy('id')
-        ->map(function ($statusGroup) {
-            return $statusGroup->sortByDesc('pivot.status_date')->first();
-        })
-        ->pluck('pivot.status_date', 'id');
-    }
-
     private function buildFileName(array $params): string
```

Comment:
`$params` is unused. Also, if this filename ends up getting used, I don't think that `ClearExports` will clean it since it does not have a `.csv` extension.